res/ui/Tactile/flow.js

```diff
@@ -47,20 +47,20 @@ function create_steps() {
      */
 
     new_scene(Tactile.Wires.RevealTheWires);
-    new_step(function(scene) {
+    new_step(function (scene) {
         scene.bg_image = 'Wires/touchscreen.png';
         scene.instructions = qsTr(
-            "There are amazing things hidden inside your screen...\n" +
+            "<p>There are amazing things hidden inside your screen...\n</p>" +
```

Comment:
Do the `<p>`s work in QML? If they do then that's awesome.

Reply:
yep it does work! I looked in the documentation.

Reply:
Amazing. Great work.